### PR TITLE
Issue-34

### PR DESCRIPTION
**Pull request by hassanmehmud**
Hardcoded messages converted to dynamic, Error cases handled for encryp/decrypt

#### What does this PR do?
Adds dynamic messages in console
Adds error case variables to API file and handles those case in Console

#### Where should the reviewer start?
Small changes made to EasyCryptoAPI.java and EasyCryptoConsole.java
:checkered_flag:

#### How should this be manually tested?
Try to reproduce mentioned issue in issue-34, you won't find it anymore.
Start console normally and encrypt and decrypt messages
upon choosing wrong method system will prompt user to enter right choice etc.

### CODE REVIEW

File: JEasyCrypto/src/easycrypto/EasyCryptoAPI.java [hunk 1]
```diff
@@ -26,7 +26,11 @@
 	public enum ResultCode {
 		ESuccess,      /*!< Text was successfully en/decrypted. */
 		EError,        /*!< There was an error with en/decryption. */
-		ENotSupported  /*!< The method of en/decription is not supported by the library */
+		ENotSupported,  /*!< The method of en/decription is not supported by the library */
+		DSuccess,	/*!<Text was successfully*/
```
Comment (a-rmz) [inline JEasyCrypto/src/easycrypto/EasyCryptoAPI.java]: Successfully what?

File: JEasyCryptoConsole/src/EasyCryptoConsole/EasyCryptoConsole.java [hunk 2]
```diff
@@ -22,32 +22,48 @@ public static void main(String[] args) {
...
 					System.out.println(result.result());
 					break;
 				}
 				case EError:
 				case ENotSupported: {
-					console.printf("Encrypted text is: %s \n", result.result());
+					console.printf("Method doesn conform Please try again: \n");
 					break;
 				}
+
```
Comment (a-rmz) [inline JEasyCryptoConsole/src/EasyCryptoConsole/EasyCryptoConsole.java]: This line is not needed.

Comment (hassanmehmud) [reply JEasyCryptoConsole/src/EasyCryptoConsole/EasyCryptoConsole.java]: Well, the readlines are already removed, can you be specific which line you are talking about?

File: JEasyCrypto/src/easycrypto/EasyCryptoAPI.java (hunk 1 above)
Comment (hassanmehmud) [reply JEasyCrypto/src/easycrypto/EasyCryptoAPI.java]: i am sorry my bad, i forgot to comment will do for sure.

File: JEasyCryptoConsole/src/EasyCryptoConsole/EasyCryptoConsole.java (hunk 2 above)
Comment (a-rmz) [reply JEasyCryptoConsole/src/EasyCryptoConsole/EasyCryptoConsole.java]: Line 52, the blank line between the closing bracket `}` and the next `case`.

**Pull request closed without merging**